### PR TITLE
Add babel-inline-import-loader in webpack

### PR DESCRIPTION
**Pull request by gchoqueux**
## Motivation
A webpack loader enabling files imported by babel-plugin-inline-import to trigger rebuilds when content changes.

### CODE REVIEW

**Review by tbroyer — COMMENTED** [review]
6 commits, 1 published version, last published 3 years ago :thinking: Not the kind of dependency I'd add in my projects…

BTW, does it play well with `babel-plugin-inline-import` that we already have in `.babelrc`? That one points to https://github.com/elliottsj/babel-inline-import-loader when using webpack.

**Comment by gchoqueux** [comment]
I had seen that it was an old fossil, I hesitated ;)
Thank you, your proposal seems more logical.

**Comment by gchoqueux** [comment]
@tbroyer
I replaced by `babel-inline-import-loader`

File: webpack.config.js [hunk 1]
```diff
@@ -29,10 +29,17 @@ babelConf.babelrc = false; // disabel babelrc reading, as we've just done it
...
 replacementPluginConf[1].replacements.find(decl => decl.identifierName === '__DEBUG__').replacement.value = debugBuild;
 
+const include = [
+    path.resolve(__dirname, 'src'),
+    path.resolve(__dirname, 'test'),
+    path.resolve(__dirname, 'utils'),
+];
+
 module.exports = {
+    context: path.resolve(__dirname),
```
Comment (zarov) [inline webpack.config.js:39]: Weird from webpack that we can't use this context everywhere :thinking:

**Review by zarov — APPROVED** [review]
This is welcome !

File: webpack.config.js (hunk 1 above)
Comment (gchoqueux) [reply webpack.config.js:39]: some options need absolute path...

**Pull request merged**